baker/postUpdatedHook.ts

```diff
@@ -129,16 +132,20 @@ const main = async (
     postSlug: string
 ) => {
     console.log(email, name, postId)
-    const slug = await syncPostToGrapher(postId)
-
-    if (BAKE_ON_CHANGE)
-        await new DeployQueueServer().enqueueChange({
-            timeISOString: new Date().toISOString(),
-            authorName: name,
-            authorEmail: email,
-            message: slug ? `Updating ${slug}` : `Deleting ${postSlug}`,
-        })
+    try {
+        const slug = await syncPostToGrapher(postId)
 
+        if (BAKE_ON_CHANGE)
+            await new DeployQueueServer().enqueueChange({
+                timeISOString: new Date().toISOString(),
+                authorName: name,
+                authorEmail: email,
+                message: slug ? `Updating ${slug}` : `Deleting ${postSlug}`,
+            })
+    } catch (err) {
+        console.error(err)
```

Comment:
I feel like we should send the error to Slack in this case?

Reply:
I agree - I'll merge this now and ship it but will create a follow up to fix this